docs/device_report.md

```diff
@@ -73,7 +73,6 @@ Overall device result FAIL
 |skip|poe.support|Other|Other||
 |skip|protocol.bacnet.pic|Other|Other|Bacnet device not found... Pics check cannot be performed.|
 |skip|protocol.bacnet.version|Other|Other|Bacnet device not found.|
-|skip|security.firmware|Other|Other|Could not retrieve a firmware version with nmap. Bacnet port could be closed or filtered|
```

Comment:
@ewRedstone This needs to be added back in:

https://travis-ci.org/arupiot/daq/jobs/580236271

Reply:
Nevermind - it was a trivial fix so I did it in the browser